### PR TITLE
Update CompareReportRenderer.java

### PR DESCRIPTION
**Pull request by sirklosterIII**
I slightly changed the style of the comparison report to make it more readable

### CODE REVIEW

**Comment by E1izabeth** [comment]
Thank you for contribution!

Could you attach a picture how does it look like now?

**Comment by sirklosterIII** [comment]
Sure! From eternal scrolls and visually losing text to vertical scroll
Previous
image
PR
image

File: plugins/org.jkiss.dbeaver.cmp.simple/src/org/jkiss/dbeaver/tools/compare/simple/CompareReportRenderer.java [hunk 1]
```diff
@@ -106,10 +106,15 @@ private void renderHeader() throws IOException
         xml.startElement("style");
         StringBuilder styles = new StringBuilder();
         styles.append("table {font-family:\"Lucida Sans Unicode\", \"Lucida Grande\", Sans-Serif;font-size:12px;text-align:left;} ");
-        styles.append(".missing {color:red;} .differs {color:red;} ");
+        //styles.append(".missing {color:red;} .differs {color:red;} ");
+        styles.append(".missing {color:blue;} .differs {color:blue;}"); //visibility!
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.cmp.simple/src/org/jkiss/dbeaver/tools/compare/simple/CompareReportRenderer.java]: Let's use different colors for missing and differs. We suppose that missing should be red and differs may stay blue.

Comment (sirklosterIII) [reply plugins/org.jkiss.dbeaver.cmp.simple/src/org/jkiss/dbeaver/tools/compare/simple/CompareReportRenderer.java]: Ready

**Review by E1izabeth — APPROVED** [review]

**Review by LonwoLonwo — APPROVED** [review]

**Comment by E1izabeth** [comment]
Passed to QA team.

**Comment by sirklosterIII** [comment]
Hi! what are the next steps to view this "enhacement" reflected in the product? thanks

**Comment by E1izabeth** [comment]
This PR hasn't been tested yet. It's awaiting for QA members to verify.

**Comment by E1izabeth** [comment]
After testing, if everything is OK, we will merge it and the changes will be included to the nearest release.

**Comment by sirklosterIII** [comment]
Thank you very much

**Comment by sirklosterIII** [comment]
Are QA people very busy aren't they?

**Comment by E1izabeth** [comment]
We're preparing to the major release now, so yes.

**Comment by LonwoLonwo** [comment]
Yeah, it is true. After the big release of 23.2.0, they will have much more free time for external PRs.

**Comment by uslss** [comment]
verified

**Review by ShadelessFox — APPROVED** [review]

**Pull request merged**